game_template/android/.gitignore

```diff
@@ -0,0 +1,13 @@
+gradle-wrapper.jar
+/.gradle
+/captures/
+/gradlew
+/gradlew.bat
+/local.properties
+GeneratedPluginRegistrant.java
+
+# Remember to never publicly share your keystore.
+# See https://flutter.dev/docs/deployment/android#reference-the-keystore-from-the-app
```

Comment:
It's SUPER interesting that the anchor in your original URL didn't work. I assume it has to do with using a flutter.dev URL rather than a docs.flutter.dev URL...

```suggestion
# For more info, check out
# [Reference the keystore from the app](https://docs.flutter.dev/deployment/android#reference-the-keystore-from-the-app).
```

Reply:
These things (like `.gitignore`) come from Flutter SDK itself. For example: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/templates/app_shared/android.tmpl/.gitignore.

I haven't touched these files. This is what all apps are created with.